Src/BootCamp.Chapter/Checks.cs

```diff
@@ -16,25 +16,29 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
+            // To do: call your implementation.
+            Program.PromptInt(message);
             return 0;
```

Comment:
You are doing the right thing (although you should call Lesson3.PromptInt which doesn't exist yet) but you do not return it! Replace the 0 with the line above. :)

Also delete the comment once you implemented the TODO task.